src/backend/base/langflow/api/v2/files.py

```
@@ -462,24 +463,26 @@
 
         # Delete the file from the storage service
-        await storage_service.delete_file(flow_id=str(current_user.id), file_name=file.path)
+        await storage_service.delete_file(flow_id=str(current_user.id), file_name=file_to_delete.path)
 
         # Delete from the database
-        await session.delete(file)
+        await session.delete(file_to_delete)
         await session.flush()  # Ensures delete is staged
-        await session.commit()  # Commit deletion
 
+        return {"detail": f"File {file_to_delete.name} deleted successfully"}
```

Ruff (0.11.9)

Consider moving this statement to an `else` block

(TRY300)